src/SvelteTable.svelte

```diff
@@ -26,9 +26,13 @@
   let searchValues = {};
   let columnByKey = {};
 
-  columns.forEach(col => {
-    columnByKey[col.key] = col;
-  });
+  let columnByKey;
```

Comment:
re-definition of `columnByKey` will cause errors

Reply:
fixed